src/components/TransactionItemRow/DataCells/ReceiptCell.tsx

```diff
@@ -36,12 +43,14 @@ function ReceiptCell({transactionItem, isSelected, style}: {transactionItem: Tra
                 backgroundStyles,
                 style,
             ]}
+            onMouseEnter={bind.onMouseEnter}
+            onMouseLeave={bind.onMouseLeave}
         >
             <ReceiptImage
                 source={source}
-                isEReceipt={transactionItem.hasEReceipt && !hasReceiptSource(transactionItem)}
+                isEReceipt={isEReceipt}
                 transactionID={transactionItem.transactionID}
-                shouldUseThumbnailImage={!transactionItem?.receipt?.source}
+                shouldUseThumbnailImage
```

Comment:
Coming from https://github.com/Expensify/App/issues/70651, we should handle the loading style for expenses that have receipt here. More details: https://github.com/Expensify/App/issues/70651#issuecomment-3298475977